src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -127,7 +127,7 @@ export function Wrap({ inputCurrency, outputCurrency }: WrapProps) {
     [inputCurrency.symbol, isWideWidget, outputCurrency.symbol]
   )
 
-  return <Caption icon={Info} caption={<Text />} />
+  return <Caption icon={null} caption={<Text />} />
```

Comment:
can we just make icon optional instead of passing null?

Reply:
good catch, it can just be omitted here